runner/cli.ts

```diff
@@ -30,21 +31,21 @@ var updateNotifier;
   }
 })();
 
-function run(env, args, output, callback) {
-  var done = wrapCallback(output, callback);
+export function run(env: void, args: string[], output: NodeJS.WritableStream, callback: (err: any) => void) {
```

Comment:
It's not used anywhere. Changed to `any`, as that's the type of `process.env` which is how it's called.